scripts/install

```diff
@@ -99,8 +98,13 @@ ynh_psql_setup_db --db_user=$db_user --db_name=$db_name
 ynh_script_progression --message="Setting up source files..." --weight=3
 
 ynh_app_setting_set --app=$app --key=final_path --value=$final_path
-# Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir="$final_path"
+# Note: We use git instead of ynh_setup_source, cause this repo use submodules
+git clone https://github.com/misskey-dev/misskey.git "$final_path" --quiet
```

Comment:
Can you elaborate on why you're switching to git clone x_x

This yield like a new shitload of issue and creates heterogenous practices instead of sticking to `ynh_setup_source` ...

Reply:
Because of this:
> git submodule update --init should be added to fetch fluent-emojis and missky assets

Suggested by @Lastorder-DC.

I'm still wondering if there is another way around, because as you said this will create a lot of extra work.

Reply:
One way is fork misskey and instead of submodule, add all files to repo(like before)
Someone should maintain fork, but when updating testing branch maintainer can update fork and apply

Reply:
But those file aren't stored anyway ? No way to just download them ?

Reply:
No.

Reply:
Or generated ?

I don't get it, those files are coming from somewhere, why can't we retrieve this ?

Reply:
It can be fetched from repo

https://github.com/misskey-dev/emojis/tree/cae981eb4c5189ea9ea3230e83b876a5068df7d1
https://github.com/misskey-dev/assets/tree/0179793ec891856d6f37a3be16ba4c22f67a81b5

so maybe, we can download zip instead and place to proper folder to remove git clone / submodule init stuff.(I'm not expert to git stuff so correct me If I'm wrong)

Reply:
Great ! Then we found another way, thanks :)

Now we just need to download those 2 folders https://github.com/misskey-dev/emojis/tree/cae981eb4c5189ea9ea3230e83b876a5068df7d1/dist & https://github.com/misskey-dev/assets/tree/0179793ec891856d6f37a3be16ba4c22f67a81b5/live2d/ai and place them where they are supposed to be - where is it ? 😅

@oufmilo do you think you could try to revert to the classic Yunohost install process and then download those files ?

Reply:
to misskey root folder
misskey-dev/emojis -> fluent-emojis
misskey-dev/assets-> misskey-assets

whole repo should be put there so there will be files like `misskey-assets/live2d/ai/~~~`